proto/contract.proto

```diff
@@ -105,6 +147,9 @@ message Egress {
   // activities such as:
   // - tagging metrics
   Reference reference = 11;
+
+  // CNCF CloudEvents SubscriptionsAPI compliant filters
```

Comment:
I like this clarification

would you mind adding a sentence about the "established" filters here:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/2664511176fd0f74d489700a3eafcc95173704b3/proto/contract.proto#L127

So they get a bit of context too, while introducing the new type?